### DISCUSSION TITLE
why there is no kali-dark color scheme in v2.1.0?  (kali linux)

### DISCUSSION THREAD

**Discussion by coolORANGE3** [lxqt/lxqt#2666] [Q&A]
~~ why there is no kali-dark theme in v2.1.0? should i install the previous version just for kali-dark theme? 
~~ is there a way to add kali-dark theme to the latest version of qterminal which is 2.1.0?
~~ i have been try to install qterminal 1.4.0 through .deb file, after installing the qterminal_v1.4.0.deb in my kali linux i encountered new problem and i should fix it by run "sudo apt-get update -f" and after that boom! i got latest version of qterminal which is 2.1.0 instead of 1.4.0, pleae help me guys :'<

edit: i have been try to install the qterminal v2.1.0 manually like using cmake but i found branching problems.
![image](https://github.com/user-attachments/assets/f69af9a5-a7c3-4767-bc05-0fab5785046a)

i want to have kali dark color scheme in the preferences of qterminal (this one have kali-dark color scheme because it's the previous version which 1.4.0)
![image](https://github.com/user-attachments/assets/c58851ec-d117-4b2f-9bb5-4c7d135c996a)

**Comment by tsujan** [comment]
Why do you ask it here? Doesn't Kali have a place for getting feedback from its users?

**Reply by coolORANGE3** [reply]
how?, do you mean like this "sudo apt-get install qterminal=v1.4.0" something like that?

**Reply by tsujan** [reply]
Your question is about Kali — LXQt has never had a "Kali theme" — so, you should ask it in a Kali forum.

**Reply by coolORANGE3** [reply]
sorry kali theme that i mean is color scheme in the preferences, like this, see there is no kali-dark option
![image](https://github.com/user-attachments/assets/c1b65226-29a3-40f4-a897-bcb57684d482)

**Reply by tsujan** [reply]
As I said, QTerminal never had a "Kali" scheme. The Kali package maintainers might have added it to their package. That's a downstream decision and not related to LXQt.

**Reply by coolORANGE3** [reply]
qterminal have color scheme bro , look
![image](https://github.com/user-attachments/assets/e75a5a30-9831-41cb-ac02-88e71e0fc52d)

**Reply by stefonarch** [reply]
That is added by kali, not by us, please stop bother us with it.
 Default color themes are those here:
 
 
![immagine](https://github.com/user-attachments/assets/2b8ce9d2-2b12-4185-a0ab-7c87377d0d45)

**Reply by coolORANGE3** [reply]
really? but u know what bro i though my kali-dark.colorscheme were deleted but when i try to check it in /usr/share/qtermwidget6/color-scheme/ and i found it so the question is if there is kali-dark.colorscheme then why in the preferences kali-dark it's not there?, is it really cause this v2.1.0 in based doesn't have kali-dark?


i even try to see the responds by deleted the directory of the color-schemes but after that nothing happend and i still can choose color-schemes, this make me more confused :<
![image](https://github.com/user-attachments/assets/472d2e70-9119-4665-9361-0c3d7b598976)

**Reply by tsujan** [reply]
> i try to check it in /usr/share/qtermwidget6/color-scheme/

The scheme folder is `/usr/share/qtermwidget6/color-schemes`, not `/usr/share/qtermwidget6/color-scheme/`. Ask Kali maintainers about it.

Closing this…
